metrics/metrics.go

```diff
@@ -69,6 +69,7 @@ type Metrics interface {
 	MeasureBackend5xx(t time.Time)
 	IncErrorsStreaming(routeId string)
 	RegisterHandler(path string, handler *http.ServeMux)
+	UpdateGauge(key string, value float64)
```

Comment:
this will break prometheus provider, please implement a dummy in the prometheus provider, too

Reply:
Good catch!

Reply:
Sorry I got confused. It is already implemented https://github.com/zalando/skipper/blob/master/metrics/prometheus.go#L297